deckhouse-controller/crds/doc-ru-module.yaml

```
@@ -49,3 +49,9 @@ spec:
                       description: Требование к статусу установки кластера (только для встроенных модулей DKP).
                     modules:
                       description: Список других включенных модулей, которые необходимы для работы модуля.
+              accessibility:
+                type: object
+                description: Настройки доступности.
+                properties:
+                  editions:
+                    description: Настройки изданий.
```

```suggestion
                    description: Данные о доступности модуля в редакциях Deckhouse.
```

@ipaqsa Will there only be data on DKP editions or will there be editions within the Deckhouse product used? For example, when using the module with DVP, ee will mean DVP EE or  DKP EE?